daprdocs/content/en/reference/components-reference/supported-bindings/servicebusqueues.md

```diff
@@ -153,6 +153,36 @@ curl -X POST http://localhost:3500/v1.0/bindings/myServiceBusQueue \
 
 {{< /tabs >}}
 
+## Schedule a message
+
+A message can be scheduled for delayed processing.
+
+To schedule a message use the `metadata` section in the request body during the binding invocation: the filed name is `ScheduledEnqueueTimeUtc`.
```

Comment:
```suggestion
To schedule a message, use the `metadata` section in the request body during the binding invocation. The filed name is `ScheduledEnqueueTimeUtc`.
```

Reply:
Thanks. Adjusted the change accordingly.